src/main/resources/templates/addRemoveQuestions.html

```diff
@@ -18,8 +18,6 @@ <h1>Add or remove questions</h1>
             <option value="range">Range</option>
             <option value="multipleChoice">Options</option>
         </select><br><br>
-        <label for="questionText">Create new question:</label>
-        <textarea id="questionText" name="questionText" required></textarea><br><br>
```

Comment:
Why is this removed? This causes an error since we can't choose the name of a question we add.